Makefile

```diff
@@ -36,6 +35,14 @@ hvs:
 	makeself installer deployments/installer/$*-$(VERSION).bin "$* $(VERSION)" ./install.sh
 	rm -rf installer
 
+flavorgen-installer:
+	rm -rf /opt/flavorgen
```

Comment:
Please build components relative to the source tree (ex. in the "build" dir).  Don't install them on the developers computer in /opt/flavorgen.

Reply:
If we run the exe from deployments/installer folder, error will be thrown if we try to read flavor schemas from the source tree. So we copied the schemas into the local folder.
Can we create 'build' directory under root folder and copy the flavor gen tool schemas under this folder?

Reply:
We should treat flavor-gen just like every other component that we build (ex. hvs, aas, cms, etc.).  "make flavor-installer" should result in a makeself installer that deploys the needed files onto a user's computer. 

All of the "installer" make commands copy data to local directory relative to the source tree (not to the developer's /opt directory).  It then calls the makeself to package the files into a single installer.  The installer is then responsible for deploying the executable, schema files, etc. into /opt on the user's machine.